CITi-UFPE/hackapride_2022
# Feature/navbar

# What I did?

- Navbar

## How to test?

1. Open the terminal and go to project folder

2. In terminal:
   2.1. `git checkout develop`
   2.2. `git pull`
   2.3. `git checkout Feature/Navbar`
   2.4. `git pull origin Feature/Navbar`
   2.5. `yarn install` ou `npm install`
   2.6. `code .`
   2.7. `yarn start` ou `npm start`

3. In browser go to http://localhost:3000
4. Check the navbar
